pages/settings.vue

```diff
@@ -46,13 +46,13 @@
       <div class="w-10 flex justify-center" @click="toggleJumpBackwards">
         <span class="material-icons text-4xl">{{ currentJumpBackwardsTimeIcon }}</span>
       </div>
-      <p class="pl-4">{{ $strings.LabelJumpBackwardsTime }}</p>
+      <p class="pl-4">{{ $strings.LabelJumpBackwardsTime }}: {{ settings.jumpBackwardsTime }}</p>
```

Comment:
This will need to be changed to match other localized strings that have numbers (such as below)

`  "LabelNumEpisodesIncomplete": "{0} episodes, {1} incomplete",`

Reply:
Thanks, I thought of that but I didn't want the display to wrap. I could add "sec" to it, would that be good?

Reply:
Yeah, even just an `s` would be good so an `m` could be used for minutes whenever that happens (I'm not saying you need to add the minutes as well in this PR, just making it easier to add later)

Reply:
Great, I'll do that now, thanks! Not sure how to localize those, though...

Reply:
Done, how does that look?

Reply:
It should all be in the same string instead of concatenating multiple strings. The `{0}` allows for string substitution, so if you search for a label in the codebase you can see how it is used